tests/tracing/bug01732-001.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Test for bug #1732 - Trace flamegraph format output for time
```

Comment:
I think it would be nicer if these tests were not named or described as a bug, but rather as `flamegraph-001.phpt` and described as `Tracing: Flamegraph for time`, or something like that.

Reply:
I was unsure, since your documentation tells to name files bug-00XXXX.phpt for bugs but it doesn't cover the case of new features.